packages/jest-runner/src/messaging.ts

```diff
@@ -8,14 +8,14 @@ class State {
   }
 
   public coverageAnalysis: CoverageAnalysis = 'off';
-  private mutantCoverageHandler: MutantCoverageHandler;
+  private mutantCoverageHandler?: MutantCoverageHandler;
```

Comment:
again optional? 🤔

Reply:
Yeah, the same reasoning as before. This is because I enabled `--strictPropertyInitialization`.